documentation/ingestion_server/guides/troubleshoot.md

```diff
@@ -0,0 +1,59 @@
+# Troubleshooting
+
+This guide describes various manual steps to troubleshoot issues with the
+ingestion server's processes like database transfer, ES indexing.
+
+## Interrupt indexing
+
+The ingestion server performs indexing using indexer workers, whose primary
+purpose it is to create documents from the API database and index them in
+Elasticsearch.
+
+They are EC2 instances that are stopped by default when indexing is not taking
```

Comment:
```suggestion
Indexer workers are EC2 instances that are stopped by default when indexing is not taking
```
A little redundant, but to make sure that it's clear what we're talking about :)

Reply:
Resolved in 128da0b.